docfx_project/ziti/config-store/overview.md

```diff
@@ -4,10 +4,20 @@ Ziti has the capability to define a configuration schema for an application and
 ## Why Centralized Configuration?
 One might ask why have this feature in Ziti when applications can store configuration data in local configuration files. While this approach works, centralized management makes deployments much easier. It can be difficult or impossible to update a file on a device out in the field, whereas updating the configuration in Ziti is easy, and running clients will quickly be notified of service changes.
 
-The Ziti tunneler applicatons provide an example how configuration data can be used. 
+The Ziti tunneler applicatons provide an example how configuration data can be used through standard `config-types`.
 
-* Tunnelers need to know what ip/dns and port(s) to intercept for services they are proxying on the client side
-* Tunnelers need to know where to reach out to applications they are proxying on the server side
+* Tunnelers need to know what ip/dns and port(s) to intercept for services they are proxying on the client side. For this a tunneler may use standard config type `intercept.v1`. Here's one example of a config of standard config type `intercept.v1`.
+
+    ```json
+    {"protocols":["tcp"],"addresses":["acme.example.ziti"], "portRanges":[{"low":5000, "high":5000}]}
```

Comment:
I really don't like calling these out in this way. I understand the thought, but instead, I'd rather see an issue put in to "document the 5 internal/standard configs" and i'd like to see them fully documented. this feels like debt that will go stale. instead a whole sub page about each is a better approach imo

Reply:
https://github.com/openziti/ziti-doc/issues/160